sysbox-in-docker/Dockerfile.ubuntu-focal

```diff
@@ -72,4 +70,9 @@ ENV \
 COPY s6-services /etc/services.d/
 COPY sysbox /etc/cont-init.d/
 
+# Copy Sysbox artifacts.
```

Comment:
Not sure why you moved these COPY instructions below, but if you do, please also modify the other Dockerfiles for consistency's sake.

Reply:
Also, please remember to update the images when done (it may take a few hours depending on how many layers you are touching with these changes):

```
docker push nestybox/sysbox-in-docker:ubuntu-focal && docker push nestybox/sysbox-in-docker:ubuntu-bionic && docker push nestybox/sysbox-in-docker:fedora-31 && docker push nestybox/sysbox-in-docker:fedora-32 && docker push nestybox/sysbox-in-docker:centos-8
```

[ leave debian ones out; need to fix something to make them work well ]

Reply:
I think host artifacts should be copied after all other software is installed in the container. That way the layers remain more stable. I'll update the other Dockerfiles and upload the images.